scripts/velocity.py

```diff
@@ -0,0 +1,391 @@
+
+    print(f'''## Pull requests
+
+At the time this report was generated, {len(prs)} were marked merged or \
+closed in the reporting period. Not all closed PRs are merged; some PRs \
+may have been superseded by another PR and thus closed without merging.
+
+| Reviewer | Time spent | Lines reviewed | Ratio (sec/line) |
+| -------- | ---------- | -------------- | ---------------- |''')
+    for reviewer, rtime in review_time.items():
+        print('',
+              f'[@{reviewer}](https://github.com/{reviewer})',
+              str(rtime),
+              str(review_lines[reviewer]),
+              f'{(rtime * 60 * 60) / review_lines[reviewer]:.3f}',
+              '',
+              sep='|')
+
+    merged = [p for p in prs if p.merged]
+    closed = [p for p in prs if not p.merged]
+    print(f'''
+At the time this report was generated, {len(merged)} pull requests were merged:
+''')
+    print(print_prs(*merged))
+    print(f'''
+Of the {len(prs)} pull requests closed in this reporting period, {len(closed)} \
+pull requests were closed but not merged. Points for these PRs are not \
+counted towards contributor totals above.
+''')
+    print(print_prs(*closed))
+
+    print(f'''## Resolved issues
+At the time this report was generated, {len(issues)} were marked closed in \
+the reporting period. This indicates that the associated work is considered \
+to have "landed" and been demoed.
+
+| Assignees | Issue | Estimate |
+| --------- | ----- | :------: |''')
+    for issue in issues:
+        authors = [f'[@{a.login}]({a.html_url})' for a in issue.assignees]
+        cols = [', '.join(authors), issue.markdown(), str(issue.estimate or '')]
+        print('', *cols, '', sep='|')
+
+
+@functools.lru_cache
+def get_user_by_id(user_id: int) -> str:
+    # ZenHub identifies users only by their GitHub user ID. The only way to map
+    # a user ID to a user name is to use an undocumented API endpoint that
+    # PyGithub does not directly support. (PyGithub/PyGithub#1615)
+    r = requests.get(f'https://api.github.com/user/{user_id}',
+                     auth=(config['github_user'], config['github_key']))
+    r.raise_for_status()
+    return r.json()['login']
+
+
+def _human_readable_date(since: datetime, until: datetime) -> str:
+    """
+    >>> _human_readable_date(datetime.now() - timedelta(days=25), datetime.now())
+    'the last 3 weeks and 4 days'
+    """
+    assert until.date() == datetime.today().date()
+    days = (until - since).days
+    if days == 1:
+        return 'yesterday'
+    elif days < 7:
+        return f'the last {days} days'
+    elif days == 7:
+        return 'the last week'
+    elif days == 8:
+        return 'the last eight days'
+    elif days % 7 == 0 and days > 7:
+        return f'the last {int(days / 7)} weeks'
+    elif days % 7 == 1 and days > 7:
+        return f'the last {days // 7} weeks and 1 day'
+    else:  # elif days % 7 > 1 and days > 7
+        return f'the last {days // 7} weeks and {days % 7} days'
```

Comment:
```suggestion
    elif days % 7 == 0:
        return f'the last {int(days / 7)} {week_word}'
    elif days % 7 == 1:
        return f'the last {days // 7} {week_word} and 1 day'
    else:  # elif days % 7 > 1
        return f'the last {days // 7} {week_word} and {days % 7} days'
```
Using `week_word` would give `the last 1 week and 3 days` vs `the last 1 weeks and 3 days`.
Also `days > 7` is redundant due to previous conditions in the elif tree